src/components/SelectionList/selectionListPropTypes.js

```diff
@@ -1,203 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
@shubham1206agra When migrating this file to typescript, did the author forget to delete this file?

Reply:
It is https://github.com/Expensify/App/pull/34794